src/main/kotlin/mate/academy/model/Circle.kt

```diff
@@ -0,0 +1,11 @@
+package mate.academy.model
+
+import mate.academy.service.Figure
+
+class Circle(override val color: Color, private val radius: Double): Figure {
+
+    override fun getArea(): Double = Math.PI * radius * radius
+
+    override fun draw() = println("Figure: circle, area: ${getArea()} sq. units," +
```

Comment:
You may use triple quote marks to avoid concatenation
https://kotlinlang.org/docs/strings.html#multiline-strings